src/test/java/duke/command/AddCommandTest.java

```diff
@@ -0,0 +1,38 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.*;
+import duke.ui.Ui;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.format.DateTimeParseException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class AddCommandTest {
+    @Test
+    public void InvalidDate() {
```

Comment:
Would it help with clarity if you named it following the suggested structure "featureUnderTest_testScenario_expectedBehavior()" as per the coding standard for test cases?